### PR TITLE
CSIP6: Create testCase.xml

### PR DESCRIPTION
**Pull request by koit**

### CODE REVIEW

**Review by PhillipTommerholt — APPROVED** [review]
It looks good.
You are talking about making changes to the extension schema changing xsd:string to xsd:anyURI. Could it be an idea to also make the @profile attribute mandatory in that schema? 
 I agree that A2 should have a discussion about requirements if it is a valid profile.

File: corpora/csip/metadata/mets-root/CSIP6/testCase.xml [hunk 1]
```diff
@@ -1,23 +1,95 @@
-<!-- Root element for an individual test case, allows these to be wrapped into
-XML lists -->
-<testCase xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="testCase.xsd"
-  testable="UNKNOWN">
-  <!-- Unique ID for the test case -->
-  <id specification="E-ARK CSIP" version="2.0-DRAFT" requirementId="CSIP6"/>
-  <!-- URL references to requirements for convenient lookup -->
+<testCase xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="testCase.xsd" testable="TRUE">
+  <id specification="CSIP" version="2.0.3" requirementId="CSIP6"/>
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/mets-root/CSIP6/testCase.xml]: 2.0.4 is the latest

**Review by PhillipTommerholt — CHANGES_REQUESTED** [review]
new version of CSIP is 2.0.4

File: corpora/csip/metadata/mets-root/CSIP6/testCase.xml (hunk 1 above)
Comment (koit) [reply corpora/csip/metadata/mets-root/CSIP6/testCase.xml]: Fixed.

**Review by carlwilson — APPROVED** [review]

**Pull request merged**